packages/utils/ast-utils.js

```diff
@@ -507,6 +507,62 @@ function parseMerge(j, ast, value, action) {
 		return ast;
 	}
 }
+
+/**
+ *
+ * Recursively updates an object/property to a node
+ * @param {any} j — jscodeshift API
+ * @param {Node} p - AST node
+ * @param {String} key - key of a key/val object
+ * @param {Any} value - Any type of object
+ * @returns {Node} - the updated ast
+ */
+
+function updateProperty(j, p, key, value) {
+	if (!p) {
+		return;
+	}
+	let valForNode;
+	if (Array.isArray(value)) {
+		let arrExp = j.arrayExpression([]);
+		if (safeTraverse(p, [
```

Comment:
These specific parts could be abstracted. One function to  have a sanity check for the node, return `p.value.value.type` from the function and use that. i.e `safeTraverseAndGetType(p) <string>`